data/json/items/armor/helmets.json

```
@@ -970,18 +970,17 @@
     "id": "xl_helmet_kabuto",
     "type": "ARMOR",
     "copy-from": "helmet_kabuto",
-    "name": { "str_sp": "XL kabuto" },
+    "name": { "str": "kabuto" },
```

```suggestion
    "name": { "str_sp": "kabuto" },
```

Japnaese words are usually used with the same plural form as the singular form.